models/users.js

```diff
@@ -53,6 +54,7 @@ const addOrUpdate = async (userData, userId = null) => {
         isNewUser: false,
         userId: user.docs[0].id,
         incompleteUserDetails: user.docs[0].data().incompleteUserDetails,
```

Comment:
what if docs is null

Reply:
This change is not part of my PR.

Reply:
> what if docs is null

https://github.com/Real-Dev-Squad/website-backend/blob/2dc69673d6bb4899a250425a75827381954308de/models/users.js#L50C1-L51C1

Reply:
userId: user?.docs[0]?.id,